episodes/02-practice-learning.md

```diff
@@ -93,6 +93,8 @@ for how we teach.
 
 :::::::::::::::::::::::::::::::::::::  callout
 
+## Models are not perfect but are still helpful
```

Comment:
```suggestion
### Models are not perfect but are still helpful
```

I think this is better structurally for eg screen readers, but waiting on confirmation of style